deployment/deploy-dev

```diff
@@ -12,9 +12,9 @@ WEBSITE=${DEPLOYMENT}/../website
 cd ${WEBSITE}
 npm run build:prod
 
-# deploy
-rsync -a --delete --rsync-path='sudo rsync' build/ ${USER}@${HOST}:/var/www/html/beta
-rsync -a --delete --exclude="/uploads" --exclude="/downloads/*" --rsync-path='sudo rsync' django/ ${USER}@${HOST}:/var/www/backend/beta/django
+# deploy (not preserving owner/group)
+rsync -rlptD --delete --rsync-path='rsync' build/ ${USER}@${HOST}:/var/www/html/beta
```

Comment:
curious -- how does this work?

I noticed in the docs ` -p, --perms                 preserve permissions`

Reply:
`-p` are the file permissions like `rwxr-xr-x`.
According to the man page `-a` corresponds to `-rlptgoD`. But since keeping the owner and group creates some havoc, I've removed the `go` part.

Reply:
👍